packages/dayjs/index.ts

```diff
@@ -1,6 +1,8 @@
 import dayjs from "dayjs";
 import relativeTime from "dayjs/plugin/relativeTime";
+import utc from "dayjs/plugin/utc";
```

Comment:
@PJColombo let me know what you think of this change. During a tests I found out that we were using the local time instead.

Reply:
Cool. Nice catch 👍 !